packages/pyolite-kernel/src/worker.ts

```diff
@@ -327,26 +309,27 @@ self.onmessage = async (event: MessageEvent): Promise<void> => {
 
   const messageType = data.type;
   const messageContent = data.data;
+  const parentHeader = data.parentHeader;
```

Comment:
Maybe we could do `kernel._parent_header = data.parentHeader` here instead, and not change the signature of `execute`, `complete` etc

Reply:
Done!